doc/cc65.sgml

```diff
@@ -709,7 +709,7 @@ This cc65 version has some extensions to the ISO C standard.
         places.
         <p>
 
-<item>  There are two pseudo variables named <tt/__AX__/ and <tt/__EAX__/.
+<item>  There are two pseudo variables named <tt/__A__, __AX__/ and <tt/__EAX__/.
```

Comment:
This isn't correct. However, I'll fix it myself...

Reply:
ok. thanx :)

Reply:
Done: https://github.com/cc65/cc65/commit/a861d84011df82ac5f81a8c87ae3181fee393993